pkg/virt-operator/util/config.go

```diff
@@ -99,6 +101,9 @@ type KubeVirtDeploymentConfig struct {
 
 	// environment variables from virt-operator to pass along
 	PassthroughEnvVars map[string]string `json:"passthroughEnvVars,omitempty" optional:"true"`
+
+	// generic mapping for the controller deployments flags
+	ControllerFlags map[string]string `json:"virtControllerFlags,omitempty" optional:"true"`
```

Comment:
can we call this VirtControllerFlags to align with the json field. That's a convention we generally try to adhere to.